### ISSUE TITLE
[documentation] Not described value was used in example

### ISSUE THREAD

**Issue by EugenKon** [hashicorp/nomad#18284]
### Proposal
![image](https://github.com/hashicorp/nomad/assets/139780495/5ffc90ae-a813-4c9b-b914-2612a8b66249)

I can not understand what 'default' mean. Does it have special meaning or just usual name like 'dc1'?

**Comment by lgfa29** [comment]
That's right, `default` it's just an example, there's no specific meaning to it. Since `datacenters` is now an optional field I've removed them for these examples on https://github.com/hashicorp/nomad/pull/18289. Thanks!
